cmd/lotus-shed/main.go

```diff
@@ -87,6 +87,7 @@ func main() {
 		gasTraceCmd,
 		replayOfflineCmd,
 		indexesCmd,
```

Comment:
I don't think there's anything in `indexesCmd` that needs to be kept now, delete it and `indexes.go`?

Reply:
This is done.